packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -610,6 +826,71 @@ export abstract class BaseService extends Resource
     this.node.defaultChild = this.resource;
   }
 
+  /**
+   *   Enable Deployment Alarms which take advantage of arbitrary alarms and configure them after service initialization
+  */
+  public enableDeploymentAlarms(alarmConfig: DeploymentAlarmConfig) {
+    // Throw an error if deployment alarms are already configured
+    if (this.deploymentAlarms) {
+      throw new Error('Deployment alarms are already configured.');
+    }
```

Comment:
why do we throw this error instead of letting them to override?

Reply:
Yeah, that true we can do that.
I just removed that error, so it can override